portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -64,36 +80,18 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     int maxComments = Integer.parseInt(request.getParameter(COMMENT_AMOUNT));
-    String sort = request.getParameter(SORT);
-
-    Query query = new Query(COMMENT_ENTITY);
-    if (sort.equals(OLDEST_FIRST)) {
-      query.addSort(COMMENT_TIMESTAMP, SortDirection.ASCENDING);
-    } else if (sort.equals(NEWEST_FIRST)) {
-      query.addSort(COMMENT_TIMESTAMP, SortDirection.DESCENDING);
-    } else if (sort.equals(LONGEST_FIRST)) {
-      query.addSort(COMMENT_LENGTH, SortDirection.DESCENDING);
-    }
-
+    Query query = sortedQuery(request);
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery results = datastore.prepare(query);
 
+    // TODO: Create helper method for creating all comments.
```

Comment:
Can this be removed?

Reply:
I'm planning on creating this helper method later on. Is it okay to leave it or should I resolve my TODOs before submitting a PR?

Reply:
Ah ok, no keep it. I just thought createComment() was that helper.